cards/src/main/resources/cards/custom/group0/other_death_knight/msg/spell_continuity.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Continuity",
+  "baseManaCost": 5,
+  "type": "SPELL",
+  "heroClass": "BROWN",
+  "rarity": "FREE",
+  "description": "Give a minion and all other copies of it +2/+2 (wherever they are)",
+  "targetSelection": "MINIONS",
+  "spell": {
+    "class": "MetaSpell",
+    "spells": [
+      {
+        "class": "BuffSpell",
+        "target": "ALL_MINIONS",
+        "value": 2,
+        "filter": {
+          "class": "SpecificCardFilter",
+          "secondaryTarget": "TARGET"
+        }
+      },
+      {
+        "class": "BuffSpell",
+        "target": "FRIENDLY_HAND",
+        "value": 2,
+        "filter": {
+          "class": "SpecificCardFilter",
+          "secondaryTarget": "TARGET"
+        }
+      },
+      {
+        "class": "BuffSpell",
+        "target": "FRIENDLY_DECK",
+        "value": 2,
+        "filter": {
+          "class": "SpecificCardFilter",
+          "secondaryTarget": "TARGET"
+        }
+      },
+      {
+        "class": "AddEnchantmentSpell",
```

Comment:
Maybe implement this as an enchantment with an aura that buffs the baked-in card. You'll need a custom AddEnchantmentSpell that bakes in the card that way. See `AddDeathrattleSecondaryAsTargetSpell` for an idea for how that would work.